src/Compilers/CSharp/Portable/Lowering/StateMachineRewriter/StateMachineTypeSymbol.cs

```diff
@@ -93,6 +93,8 @@ public sealed override ImmutableArray<CSharpAttributeData> GetAttributes()
             return _attributes;
         }
 
+        public override bool AreLocalsZeroed => KickoffMethod.AreLocalsZeroed;
```

Comment:
>public override bool AreLocalsZeroed => KickoffMethod.AreLocalsZeroed; [](start = 8, length = 70)

sealed?

Reply:
What is the reasoning for using `sealed` so often in this type?

Reply:
> What is the reasoning for using sealed so often in this type? 

To make the intent clear and prevent others from overriding this implementation by accident. Also, in some rare cases compiler might generate more efficient code

---
In reply to: [172899392](https://github.com/dotnet/roslyn/pull/25276#discussion_r172899392) [](ancestors = 172899392)